config/config.py.template

```diff
@@ -456,6 +458,23 @@ MLLIB_TESTS += [("naive-bayes", MLLIB_PERF_TEST_RUNNER, SCALE_FACTOR,
     MLLIB_JAVA_OPTS, [ConstantOption("naive-bayes")] +
     NAIVE_BAYES_TEST_OPTS)]
 
+if MLLIB_SPARK_VERSION >= 1.4:
+    NAIVE_BAYES_TEST_OPTS_BERNOULLI = MLLIB_REGRESSION_CLASSIFICATION_TEST_OPTS + [
+        # Expected fraction of examples which are negative
+        OptionSet("per-negative", [0.3]),
+        # The scale factor for the noise in feature values
+        OptionSet("scale-factor", [1.0]),
+        # Naive Bayes smoothing lambda.
+        OptionSet("nb-lambda", [1.0]),
+        # MLLIB_REGRESSION_CLASSIFICATION_TEST_OPTS + [
+        OptionSet("model-type", ["Bernoulli"]),
```

Comment:
Should we add `OptionSet("model-type", ["Multinomial"])` to the multinomial naive Bayes test? Otherwise it will break the test because `MODEL_TYPE` is not there.